packages/dds/tree/src/feature-libraries/forest-summary/forestSummarizer.ts

```diff
@@ -34,24 +38,36 @@ import type {
 	SummaryElementStringifier,
 } from "../../shared-tree-core/index.js";
 import { idAllocatorFromMaxId } from "../../util/index.js";
-import { chunkFieldSingle, defaultChunkPolicy } from "../chunked-forest/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { chunkFieldSingle, defaultChunkPolicy } from "../chunked-forest/chunkTree.js";
 import type { FieldBatchCodec, FieldBatchEncodingContext } from "../chunked-forest/index.js";
 
 import { type ForestCodec, makeForestSummarizerCodec } from "./codec.js";
 import type { Format } from "./format.js";
+import { ForestIncrementalSummaryBuilder } from "./incrementalSummaryBuilder.js";
+import { TreeCompressionStrategyExtended } from "../treeCompressionUtils.js";
+
 /**
- * The storage key for the blob in the summary containing tree data
+ * The key for the blob in the summary containing the forest's contents.
  */
-const treeBlobKey = "ForestTree";
+export const forestSummaryContentKey = "ForestTree";
+
+export const forestSummaryKey = "Forest";
```

Comment:
This should have docs. I don't know what it is.

Reply:
Ah, I see its document inside of ForestSummarizer.key. Splitting it out like this separates it from its documentation. Perhaps some explanation of what this means besides being ForestSummarizer.key here, and or a link or inheritdoc or moving the comment to here from ForestSummarizer.key would be appropriate.